openedx/core/djangoapps/programs/utils.py

```diff
@@ -334,31 +335,79 @@ def _attach_course_run_upgrade_url(self, run_mode):
         else:
             run_mode['upgrade_url'] = None
 
-    def _attach_instructors(self):
-        """
-        Extend the program data with instructor data. The instructor data added here is persisted
-        on each course in modulestore and can be edited in Studio. Once the course metadata publisher tool
-        supports the authoring of course instructor data, we will be able to migrate course
-        instructor data into the catalog, retrieve it via the catalog API, and remove this code.
-        """
+    def _attach_course_run_can_enroll(self, run_mode):
+        run_mode['can_enroll'] = bool(has_access(self.user, 'enroll', self.course_overview))
+
+
+# pylint: disable=missing-docstring
+class ProgramMarketingDataExtender(ProgramDataExtender):
+    """
+    Utility for extending program data meant for the program marketing page with
```

Comment:
Please make it more clear here that this is meant for an open-source marketing page hosted by the LMS. We don't want to confuse others by making them think that this is used by edx.org.